sdks/python/apache_beam/pipeline.py

```diff
@@ -152,6 +153,14 @@ def __init__(self, runner=None, options=None, argv=None):
       raise ValueError(
           'Pipeline has validations errors: \n' + '\n'.join(errors))
 
+    # set default experiments for portable runner
+    # (needs to occur prior to pipeline construction)
+    if self._options.view_as(StandardOptions).runner == 'PortableRunner':
+      experiments = (self._options.view_as(DebugOptions).experiments or [])
+      if not 'beam_fn_api' in experiments:
+        experiments.append('beam_fn_api')
+        self._options.view_as(DebugOptions).experiments = experiments
```

Comment:
Could we change the default value of experiments to `[]`? Then:

```python
if not 'beam_fn_api' in self._options.view_as(DebugOptions).experiments: 
  self._options.view_as(DebugOptions).experiments.append('beam_fn_api')
```

Reply:
Unfortunately append is on a copy and the mutated list needs to be assigned, which requires the extra lines.

Reply:
@mxm I prefer less verbosity and what you suggest is what I also tried. That's when I had to learn that unfortunately you have to assign the modified list via `self._options.view_as(DebugOptions).experiments = ...` as otherwise you have only changed a copy. I could have included the default change, although it won't cut down the lines here.

Reply:
@tweise Ah, understood! Fine then :)